pyne/mesh.py

```diff
@@ -1229,44 +1224,47 @@ def tag_cell_fracs(self, cell_fracs):
         num_vol_elements = len(self)
         # Find the maximum cell number in a voxel
         max_cell_number = -1
```

Comment:
I missed this in #929, but I recommend calling this `max_num_cells`.  This variable name implies a different meaning: the largest cell ID in the problem.